Introduction to Federated Learning with Substra.ipynb

```diff
@@ -0,0 +1,1143 @@
+{
```

Comment:
<p>Nice question :) Maybe have a screenshot of a mnist worklfow to show them ? (I you haven't already thought about it)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Substra/substra-workshops/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>